src/coreclr/tools/StressLogAnalyzer/StressLogDump.cpp

```diff
@@ -183,13 +181,17 @@ void formatOutput(struct IDebugDataSpaces* memCallBack, ___in FILE* file, __inou
                                     MethodDescData.Request(g_sos,(CLRDATA_ADDRESS)arg);
 
                                     static WCHAR wszNameBuffer[1024]; // should be large enough
-                                    if (g_sos->GetMethodDescName(arg, 1024, wszNameBuffer, NULL) != S_OK)
+                                    static char szNameBuffer[ARRAY_SIZE(wszNameBuffer) * 3];
```

Comment:
+ 1 for the buffer size?

*3 + 1 is the conversion to max length for UTF-8, right?

Reply:
Yep